core/src/main/java/org/elasticsearch/index/query/GeoBoundingBoxQueryBuilder.java

```diff
@@ -88,7 +81,7 @@ public GeoBoundingBoxQueryBuilder(String fieldName) {
      * @param right The right longitude
      */
     public GeoBoundingBoxQueryBuilder setCorners(double top, double left, double bottom, double right) {
-        if (!ignoreMalformed) {
+        if (! GeoValidationMethod.isIgnoreMalformed(validationMethod)) {
```

Comment:
nit: I started to like the `(expression == false)` notation I saw in several places.